.github/workflows/update_robottelo_image.yml

```diff
@@ -0,0 +1,52 @@
+# Update robottelo image on quay.
+name: update_robottelo_image
+
+on:
+  push:
+    branches:
+      - master
+      - 6.*.z
+  repository_dispatch:
+    types:
+      - airgun-pr-merged
+      - nailgun-pr-merged
```

Comment:
If we have cherry-picked this to 6.*.z  branches then any pr merged in the Airgun/Nailgun runs the deploying containers of all the cherrypick branches irrespective of what branch is merged. @jameerpathan111 can you please test that and see if that is not a condition